### ISSUE TITLE
Using Reynir to parse laws from Alþingi

### ISSUE THREAD

**Issue by rbjarnason** [vthorsteinsson/Reynir#45]
We at the Citizens Foundation in Iceland (Íbúar ses) are adding back an old feature to our Better Iceland website where we fetch issues from Alþingi to allow citizens to debate them. I'd be interested in using something like Reynir to help citizens understand laws better by providing some analysis and maybe key points from the law.

I wanted to know if you have had any experience using Reynir to parse laws from Alþingi?

Our citizens participation app we use for Better Iceland and Better Reykjavík is Open Source here:
https://github.com/rbjarnason/your-priorities-app

We are also working on an EU funded project called Active Citizen that aims to empower citizens democratically with AI and VR.  We see Reynir fitting into this work as well and plan experiments with a dataset from Better Reykjavík.
https://github.com/rbjarnason/active-citizen
http://www.citizens.is/active

**Comment by vthorsteinsson** [comment]
Yes, there is rudimentary - very basic - support for scraping and parsing text from the althingi.is website. See https://github.com/vthorsteinsson/Reynir/blob/master/scrapers/default.py#L789 - However, this could be improved and enhanced. PRs welcome and if you have specific requirements I can also try to help to implement them.

**Comment by rbjarnason** [comment]
@vthorsteinsson Thanks for this - sounds good :) I will do some experiments with the code in the next few weeks and get back to you.
